buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -80,6 +80,10 @@ open class BuildInfo(project: Project) {
 
   val isReleaseBuild: Boolean by lazy { java.lang.Boolean.getBoolean("releaseBuild") }
 
+  val isNativeArch: Boolean by lazy { java.lang.Boolean.getBoolean("nativeArch") }
+
+  val isEnableOracleGraalvm: Boolean by lazy { java.lang.Boolean.getBoolean("oracleGraalvm") }
```

Comment:
pass `gradlew ... -DoracleGraalvm=true` to enable [SBOM](https://www.graalvm.org/latest/reference-manual/native-image/guides/use-sbom-support/) and [G1 GC](https://www.graalvm.org/latest/reference-manual/native-image/optimizations-and-performance/MemoryManagement/); intended to guard for features which are only supported within Oracle GraalVM and not GraalVM Community Edition